components/apimgt/org.wso2.carbon.apimgt.rest.api.admin.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/admin/v1/impl/ApisApiServiceImpl.java

```diff
@@ -0,0 +1,105 @@
+package org.wso2.carbon.apimgt.rest.api.admin.v1.impl;
```

Comment:
License header is missing. Please check the other places asa well

Reply:
Fixed in https://github.com/wso2/carbon-apimgt/pull/12200/commits/a8775522eeae6a224148469405bb5cdb7c76c4c5